src/clj/game/cards/events.clj

```diff
@@ -2904,7 +2904,8 @@
      :trace {:base 3
              :unsuccessful
              {:async true
-              :msg "reveal all cards in HQ"
+              :msg (msg "reveal all cards in HQ" (when-let [hand (seq (get-in @state [:corp :hand]))]
```

Comment:
```suggestion
              :msg (msg "reveal all cards in HQ" (when-let [hand (seq (:hand corp))]
```

Reply:
Ah didn't realize `msg` also bound `corp` and `runner` for you, perfect.